From 50208eafca2b1187079c0c5c54e5cf804c0ce0d1 Mon Sep 17 00:00:00 2001
From: Hrushiraj Gandhi <hrushirajg23@gmail.com>
Date: Thu, 30 Jul 2026 10:29:46 +0530
Subject: [PATCH 09841/15333] dt-bindings: arm: rockchip: Add Vicharak Vaaman
 board

Document the compatible string for the Vicharak Vaaman, an RK3399-based
single-board computer.

Signed-off-by: Hrushiraj Gandhi <hrushirajg23@gmail.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@oss.qualcomm.com>
Link: https://patch.msgid.link/20260730045947.388660-2-hrushirajg23@gmail.com
Signed-off-by: Heiko Stuebner <heiko@sntech.de>
---
 Documentation/devicetree/bindings/arm/rockchip.yaml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/arm/rockchip.yaml b/Documentation/devicetree/bindings/arm/rockchip.yaml
index 93e7cba943fa..9c3e24d471c1 100644
--- a/Documentation/devicetree/bindings/arm/rockchip.yaml
+++ b/Documentation/devicetree/bindings/arm/rockchip.yaml
@@ -1345,6 +1345,11 @@ properties:
           - const: vicharak,axon
           - const: rockchip,rk3588
 
+      - description: Vicharak Vaaman
+        items:
+          - const: vicharak,vaaman
+          - const: rockchip,rk3399
+
       - description: Vicharak Vaaman2
         items:
           - const: vicharak,vaaman2
-- 
2.55.0

